hadoop-ozone/dist/src/main/smoketest/ozonefs/ozonefs.robot

```diff
@@ -163,6 +163,18 @@ Get file
                    Execute               ozone fs -get ${BASE_URL}${SCHEME}.txt /tmp/GET.txt
                    File Should Exist     /tmp/GET.txt
 
+Inherit Acls
+                   Execute               ozone sh bucket addacl -a=user:user1:rlw[DEFAULT] ${VOLUME}/${BUCKET}
+                   Execute               ozone fs -mkdir -p ${BASE_URL}dir1/dir2
+                   Verify ACL            key   ${VOLUME}/${BUCKET}/dir1/        USER   user1   READ WRITE LIST
```

Comment:
`Verify ACL` requires importing `shell.robot`:

https://github.com/apache/ozone/blob/8847b2c60bcdc831a7d515c414cd9c28bbf2bbf4/hadoop-ozone/dist/src/main/smoketest/basic/links.robot#L20

Reply:
`Java test` has been used instead of `robot`